pkg/security/podsecuritypolicy/provider.go

```diff
@@ -413,3 +421,20 @@ func hostPortRangesToString(ranges []policy.HostPortRange) string {
 	}
 	return formattedString
 }
+
+// validates that the actual RuntimeClassName is contained in the list of valid names.
+func validateRuntimeClassName(actual *string, validNames []string) field.ErrorList {
+	for _, valid := range validNames {
+		if actual == nil {
+			if valid == "" {
```

Comment:
drop this in favor of making admins set `defaultRuntimeClassName` if they want to require all pods to have a runtimeclass specified